### PR TITLE
Copy on windows

### PR DESCRIPTION
**Pull request by phadej**
Builds on top of #6506

### CODE REVIEW

File: Cabal/doc/nix-local-build.rst [hunk 1]
```diff
@@ -559,8 +559,9 @@ repository, this command will build cabal-install HEAD and symlink the
 
     $ cabal v2-install exe:cabal
 
-Where symlinking is not possible (eg. on Windows), ``--install-method=copy``
-can be used:
+Where symlinking is not possible (eg. on Windows) the ``copy`` method
+is used by default. You can specify the instll method
```
Comment (fgaz) [inline Cabal/doc/nix-local-build.rst]: Typo: instll

File: cabal-install/Distribution/Client/CmdInstall.hs [hunk 2]
```diff
@@ -668,8 +675,15 @@ installExes verbosity baseCtx buildCtx platform compiler
   where
     overwritePolicy = fromFlagOrDefault NeverOverwrite $
                       cinstOverwritePolicy clientInstallFlags
-    installMethod   = fromFlagOrDefault InstallMethodSymlink $
-                      cinstInstallMethod clientInstallFlags
+    isWindows = System.Info.os == "mingw32"
```
Comment (fgaz) [inline cabal-install/Distribution/Client/CmdInstall.hs]: Didn't we have a function for that?

Comment (phadej) [reply cabal-install/Distribution/Client/CmdInstall.hs]: Indeed we do.

**Comment by phadej** [comment]
This needs writing symlinking code for windows. Will do soon-ish.

**Pull request merged**